src/pages/ps_reference/changelog/index.md

```diff
@@ -8,6 +8,28 @@ description: Contains a running log of changes to the UXP API environment in Ado
 
 ## Photoshop Beta (24.2 February 2022)
 
+### UXP 6.4 Integration
+From the [UXP team](../../uxp/) we have integrated the following update.
+### New
+- Webviews for Panels
+- Support Path Module APIs
+- Script can run fsapi with full access in localFileSystem
+
+### Bugs Fixes
```

Comment:
Can we leave out the bug fixes for now? A lot of these may not be relevant for 3P developers.